fast/stages/0-bootstrap/main.tf

```
@@ -28,6 +28,12 @@ locals {
     for k, v in local.groups :
     k => "group:${v}"
   }
+  locations = local.checklist.location == null ? var.locations : {
+    bq      = var.locations.bq
+    gcs     = var.locations.gcs
+    logging = local.checklist.location
```

In places we use now `locations.logging` we had earlier `var.locations.logging`. We are now only passing value from checklist. Missing `coalesce()`?

---

why would you need coalesce? if the checklist location is null we use the var, if it's not null we use it

---

unless I did my usual hurried PR, and we are using locations.logging instead of local.locations.logging :)

---

We use `local.locations.logging` in other places now. I'd expect that with no checklist, I'd had `var.locations.logging` there.  I think this is the best place to select the one (either `var.locations.logging` or `local.checklist.location`)

---

I'm not understanding what you're saying Wiktor: we do the choice of which locations to use here, and we use this local everywhere so what is it you are not liking? I'm resolving this, feel free to reopen and make it clearer

---

I mean, if you do now:
`$ grep 'var.locations.logging' *
`
This variable is never used. 

I imagine that here it should state:
```suggestion
    logging = coalesce(local.checklist.location, var.locations.logging)
```

Otherwise we change the behavior if checklist is not provided in logging export configuration.

---

I am still not following :) If checklist is not null logging will always be provided

---

done :)